### PR TITLE
Add introduction video script

### PR DESCRIPTION
**Pull request by kaklakariada**

### CODE REVIEW

File: oft-introduction-video-script.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+spec.md
+
+```
+`feat~flashlight~1`
+Tha flashlight app lets uses the phone's flash LED as a flashlight.
+
+Needs:req
+```
+
+This is the minimal form of a requirement defined in OFT's Markdown notation. It starts with a unique ID, consisting of an artifact type prefix, a name or number and a revision number. This followed by a description. And it demands coverage in another requirement prefixed with the abbreviation `REQ`.
```
Comment (kaklakariada) [inline oft-introduction-video-script.md]:
```suggestion
This is the minimal form of a requirement defined in OFT's Markdown notation. It starts with a unique ID, consisting of an artifact type prefix, a name or number and a revision number. This followed by a description. And it demands coverage in another requirement prefixed with the abbreviation `req`.
```

File: oft-introduction-video-script.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Let's take a look at a basic example for a flashlight app on a phone and define the main feature in a simple Markdown text file.
+
+
+spec.md
+
+```
+`feat~flashlight~1`
+Tha flashlight app lets uses the phone's flash LED as a flashlight.
+
+Needs:req
```
Comment (kaklakariada) [inline oft-introduction-video-script.md]:
```suggestion
Needs: req
```

File: talk_proposal_1200_characters.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Requirements define your product. They ensure that the user needs are met.
+
+OpenFastTace (OFT) verifies the integrity of a product's requirement structure, thus avoiding the costs of common mistakes like forgetting to implement requirements or to remove obsolete ones.
+
+OFT crawls specification documents and code and correlating them with the help of special unique IDs.
```
Comment (kaklakariada) [inline talk_proposal_1200_characters.txt]:
```suggestion
OFT crawls specification documents and code, correlating them with the help of special unique IDs.
```

File: video-scripts/setup.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+1. Install Cargo using APT:
+   ```
+   sudo apt install cargo
+   ```
+2. Install Autocast with Cargo:
+   ```
+   cargo install autocast
+   ```
+4. Verify your Autocast installation by running:
```
Comment (kaklakariada) [inline video-scripts/setup.md:40]:
```suggestion
3. Add `$HOME/.cargo/bin` to your `PATH` to be able to run the installed binaries
4. Verify your Autocast installation by running:
```

File: video-scripts/setup.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+# Video Creation Setup
+
+## Prerequisites
+
+* [asciinema](https://asciinema.org/) - record console sessions through script and keystroke protocol
+* [autocast](https://github.com/k9withabone/autocast) - programmatically create a asciinema cast
+* [recordmydesktop](https://enselic.github.io/recordmydesktop/)
```
Comment (kaklakariada) [inline video-scripts/setup.md]:
```suggestion
* [kazam](https://github.com/henrywoo/kazam) - record screen to MP4
```

Comment (kaklakariada) [reply video-scripts/setup.md]: Also add prerequisites:
* `highlight` command line tool
* OFT must be available at `$HOME/git/openfasttrace/product/target/openfasttrace-3.8.0.jar`

File: video-scripts/short_introduction/oft_short_introduction.yaml [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+  - !Wait 1s
+  - !Command
+    command: '#    * define requirements'
+  - !Wait 1s
+  - !Command
+    command: '#    * on different levels'
+  - !Wait 1s
+  - !Command
+    command: '#    * trace to check if they are covered'
+  - !Wait 5s
```
Comment (kaklakariada) [inline video-scripts/short_introduction/oft_short_introduction.yaml]: Add URL to OFT

File: video-scripts/short_introduction/oft_short_introduction.yaml [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+    command: "# If we trace all three files together, the requirement chain is now complete."
+  - !Wait 2s
+  - !Command
+    command: oft trace user_requirements.md design.md square_root.py
+  - !Wait 5s
+  - Clear
+
+  - !Marker Summary
+  - !Command
+    command: '# OpenFastTrace (OFT) lets you...'
```
Comment (kaklakariada) [inline video-scripts/short_introduction/oft_short_introduction.yaml]:
```suggestion
    command: '# OpenFastTrace lets you...'
```
Either introduce abbreviation in the beginning or better omit it completely.

File: video-scripts/short_introduction/oft_short_introduction.yaml [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+  width: 120
+  height: 40
+  title: OpenFastTrace - A Short Introduction
+  timeout: 90s
+  type_speed: 50ms
+  prompt: '▶ '
+
+instructions:
+  - !Command
+    command: function show(){ echo; highlight -O truecolor --wrap-no-numbers -W --line-numbers "$@"; echo; }
```
Comment (kaklakariada) [inline video-scripts/short_introduction/oft_short_introduction.yaml]:
```suggestion
    command: function show(){ echo; highlight ‐‐out‐format=truecolor --wrap-no-numbers ‐‐wrap --line-numbers "$@"; echo; }
```

**Review by kaklakariada — COMMENTED** [review]
✅ Approved

**Review by redcatbear — APPROVED** [review]

**Pull request merged**